models/public/efficientnet-b0-pt/efficientnet-b0-pt.md

```diff
@@ -0,0 +1,78 @@
+# efficientnet-b0-pt
+
+## Use Case and High-Level Description
+
+The `efficientnet-b0-pt` model is one of the [efficientnet](https://arxiv.org/abs/1905.11946) group of models designed to perform image classification. This model was pretrained in PyTorch\* All the EfficientNet models have been pretrained on the ImageNet image database. For details about this family of models, check out the [repository](https://github.com/rwightman/gen-efficientnet-pytorch).
+
+The model input is a blob that consists of a single image of 1x3x224x224 in RGB
+order. The BGR mean values need to be subtracted as follows: [123.675,116.28,103.53]
+before passing the image blob into the network. In addition, values must be divided
+by [58.395,57.12,57.375].
+
+The model output for `efficientnet-b0-pt` is the typical object classifier output for
+the 1000 different classifications matching those in the ImageNet database.
```

Comment:
Do we even need these descriptions? They seem redundant with the Input/Output sections.

Reply:
Yes, because it summarize information about  model. I/O sections are more about technical details.

Reply:
But aren't they basically the same information with different wording?